src/common/getCols.ts

```diff
@@ -2214,6 +2215,12 @@ const cols: {
 		sortSequence: ["desc", "asc"],
 		sortType: "number",
 	},
+	"rating:ovr": {
```

Comment:
For these three duplicated columns (rating:ovr, rating:pos, rating:pot), if we do indeed need the new names, then delete the old ones and try to use the new names anywhere the old ones were used. I know it might be hard to find all of them and you might miss one, but that's fine. It'll cause an error and be noticed soon enough. That's better than keeping duplicated code around forever.

Reply:
removed them, i think i put those in early when i was still experimenting with ideas